packages/experience-builder-types/test/__fixtures__/composition.ts

```diff
@@ -0,0 +1,173 @@
+import { LATEST_SCHEMA_VERSION } from '../../src';
+import { Composition, ExperienceEntry, SchemaVersions } from '../../src/types';
```

Comment:
should we delete fixtures from this package? I don't think they belong here

Reply:
I moved our `EntityStore` which we are extending from VisualSdkEntityStore to `types` package. So is it's unit test file. That's why we need these fixtures.

Reply:
Is there a reason why we need EntityStore to be moved into the types package? It is only used by the sdk, no?

Reply:
We are using EntityStore in `Experience` type. So moved it along. Otherwise I think there will be circular dependency issue.

Reply:
But I think `Experience` uses the type from `visual-sdk`, isn't it? The EntityStore is just a build up on top of `EntityStore` class that is imported from `visual-sdk`

Reply:
`Experience` uses our `EntityStore` which we have created by extending `EntityStore` from `visual-sdk`

https://github.com/contentful/experience-builder/blob/main/packages/experience-builder-sdk/src/types.ts#L310

Reply:
In that case, can we keep the experience type on the sdk side?

in that case we might need to make Experience a generit type. I wouldn't pull the entityStore implementation into the types package

```
type Experience<T extends EntityStore> = {
   entityStore: T,
   ...
}
```

Reply:
Added these changes.